### ISSUE TITLE
Pressing back from favourites always takes back to Home section

### ISSUE THREAD

**Issue by Swati4star** [Swati4star/Images-to-PDF#721]
### Steps to reproduce
- go to any section (let's say `View Files`)
- select favourites from action bar
- press back

### Expected behaviour
It should go back to previous section (`View Files` in this case)

### Actual behaviour
It goes to `Home` section

**Comment by archi14** [comment]
@Swati4star I would like to work on this.

**Comment by sidhuparas** [comment]
Sure @archi14! Go ahead.

**Comment by vyankatesh24** [comment]
> @Swati4star I would like to work on this.

@archi14 
Aren't you working on #713 ?

**Comment by archi14** [comment]
@vyankatesh24  I am working on both. Aren't we allowed to take two at the same time?

**Comment by Swati4star** [comment]
No. You should work on one at a time. You can take up next one after creating a PR for first. 
These are easy issues also.

**Comment by archi14** [comment]
@Swati4star okay

**Comment by archi14** [comment]
@Swati4star Is it okay if I work on this now?

**Comment by Swati4star** [comment]
sure @archi14 👍

**Comment by vyankatesh24** [comment]
@archi14 Any progress?

**Comment by Ni3verma** [comment]
@vyankatesh24 if archi is not working on it, assign it to me

**Comment by vyankatesh24** [comment]
@archi14 Are you still working on it?

**Comment by vyankatesh24** [comment]
@Ni3verma Go ahead :+1:

**Comment by archi14** [comment]
@vyankatesh24 sorry had some issues with my laptop.

**Comment by Swati4star** [comment]
@archi14 Please comment on one issue at a time! We cant keep track of every participant, each time you comment on an issue! :(

**Comment by archi14** [comment]
@Swati4star I thought this one got assigned to ni3verma. I was just apologizing for not responding.
